src/pages/iou/steps/IOUAmountPage.js

```diff
@@ -147,7 +152,7 @@ class IOUAmountPage extends React.Component {
                         : ROUTES.getIouRequestCurrencyRoute(this.props.reportID))}
                     >
                         <Text style={styles.iouAmountText}>
-                            {this.props.selectedCurrency.currencySymbol}
+                            {this.props.currencyList[this.props.iou.selectedCurrencyCode].symbol}
```

Comment:
NAB / just something to think about but I would maybe suggest using `lodashGet` in a situation like this to prevent the possibility of a reference error...

```js
lodashGet(this.props.currencyList, [this.props.iou.selectedCurrencyCode, 'symbol']);
```